docs/source/release_notes/release-notes.rst

```diff
@@ -40,6 +40,10 @@ develop
     *    - Type
          - Change
 
+    *    - |changed|
+         - Default 'gc_grace_seconds' set by AtlasDB for Cassandra tables has been changed from four days to an hour.  This will allow Cassandra to start cleaning up swept data sooner after sweeping.
```

Comment:
I'd like to highlight what customers should do for their existing tables, but I may just do that in the email form instead of here.